src/appengine/handlers/cron/project_setup.py

```diff
@@ -919,47 +922,46 @@ def get(self):
       logs.log_error('Failed to get honggfuzz Fuzzer entity.')
       return
 
-    project_setup_config = local_config.ProjectConfig().sub_config(
-        'project_setup')
-    bucket_config = project_setup_config.sub_config('build_buckets')
-
-    if not bucket_config:
-      raise ProjectSetupError('Project setup buckets not specified.')
-
-    config = ProjectSetup(
-        BUILD_BUCKET_PATH_TEMPLATE,
-        REVISION_URL,
-        project_setup_config.get('build_type'),
-        segregate_projects=project_setup_config.get(
-            'segregate_projects', default=False),
-        engine_build_buckets={
-            'libfuzzer': bucket_config.get('libfuzzer'),
-            'libfuzzer-i386': bucket_config.get('libfuzzer_i386'),
-            'afl': bucket_config.get('afl'),
-            'honggfuzz': bucket_config.get('honggfuzz'),
-            'none': bucket_config.get('no_engine'),
-            'dataflow': bucket_config.get('dataflow'),
-        },
-        fuzzer_entities={
-            'libfuzzer': libfuzzer,
-            'honggfuzz': honggfuzz,
-            'afl': afl,
-        },
-        add_info_labels=project_setup_config.get(
-            'add_info_labels', default=False),
-        add_revision_mappings=project_setup_config.get(
-            'add_revision_mappings', default=False),
-        additional_vars=project_setup_config.get('additional_vars'))
-
-    projects_source = project_setup_config.get('source')
-    if projects_source == 'oss-fuzz':
-      projects = get_oss_fuzz_projects()
-    elif projects_source.startswith(storage.GS_PREFIX):
-      projects = get_projects_from_gcs(projects_source)
-    else:
-      raise ProjectSetupError('Invalid projects source: ' + projects_source)
+    project_setup_configs = local_config.ProjectConfig().get('project_setup')
```

Comment:
We probably need to change the example project.yaml file in configs/test directory ? maybe add example for suffix (commented) ?

Reply:
Hmm, I don't think we need to document this for now. Would make it more difficult to change the format (like I'm doing now) in the future. Can add this later if there is demand.